scripts/cfn

```diff
@@ -131,7 +134,11 @@ if __name__ == "__main__":
 
         if values.s3bucket:
             # Upload to S3 and create the stack
-            s3conn = boto.s3.connect_to_region(values.region)
+            s3conn = None
+            if values.s3region != values.region:
```

Comment:
I set the default to be `us-east-1`.  Thinking about it, I could just forgo this entire statement and just change the original to:

```py
s3conn = boto.s3.connect_to_region(values.s3region)
```

What do you think?  If you'd prefer it to be `None` as a default I can update that as well.